sdk/core/Azure.Core/tests/LowLevelClientTests.cs

```diff
@@ -183,10 +183,12 @@ public async Task ThrowOnErrorDoesntThrowOnSuccess()
             var mockTransport = new MockTransport(mockResponse);
             PetStoreClient client = CreateClient(mockTransport);
 
+            #region Snippet:SetRequestContext
```

Comment:
nit: I think it reads more clearly to push RequestContext down to a new line:

```c#
Response response = await client.GetPetAsync("snoopy",
    new RequestContext()
    {
        ErrorOptions = ErrorOptions.Default
    });
```

Reply:
Done. But I removed the snippet directive per comments below.